src/prefect/cli/work_queue.py

```diff
@@ -206,7 +206,10 @@ async def inspect(
 async def ls(
     verbose: bool = typer.Option(
         False, "--verbose", "-v", help="Display more information."
-    )
+    ),
+    work_queue_regex: str = typer.Option(
+        None, "--regex", "-r", help="Python regex string used to match work queue names"
```

Comment:
Maybe we can call this `--match` or `--pattern` instead? That it's a regex feels like a detail for the description.

Reply:
plus, who knows maybe one day we would decide to match differently than by regex, so good call